src/components/app.tsx

```diff
@@ -9,7 +9,10 @@ import DispenserStateManager from '../dispenser-state-manager';
 import DoorStateManager from '../door-state-manager';
 import FleetManager from '../fleet-manager';
 import LiftStateManager from '../lift-state-manager';
-import { RobotTrajectoryManager } from '../robot-trajectory-manager';
+import * as NegotiationStatusManager from '../negotiation-status-manager';
```

Comment:
You can also do this for avoid importing the whole package.
`import NegotiationStatusManager from '../negotiation-status-manager';`

and use just `NegotiationStatusManager` instead of `NegotiationStatusManager.default`

Reply:
replaced 789fed3